### PR TITLE
Remove dependency on pyfarm-master from unittests

### PR DESCRIPTION
**Pull request by opalmer**
This PR removes the requirement to run the master in order for all tests to pass.  #303 also benefits from this because it will normalize test execution on Travis and Appveyor.

### CODE REVIEW

**Comment by opalmer** [comment]
@guidow, PTAL

**Comment by opalmer** [comment]
@guidow, any comments on this one?

File: README.rst [hunk 1]
```diff
@@ -61,32 +61,32 @@ on platform)::
 Testing
 -------
 
-Tests are run on `Travis <https://travis-ci.org/pyfarm/pyfarm-agent>`_ and
-`Appveyor <https://ci.appveyor.com/project/opalmer/pyfarm-agent/history>`_ for
```
Comment (guidow) [inline README.rst]: Why are you removing references to Appveyor again?

**Comment by guidow** [comment]
Looks good to me.

File: README.rst (hunk 1 above)
Comment (opalmer) [inline README.rst]: Opps, that was not intentional. Readding.

**Pull request merged**